misc/userscripts/dmenu_qutebrowser

```diff
@@ -38,11 +38,13 @@
 #				(This is unnecessarily long. I use this rarely, feel free to make this script accept parameters.)
 #
 
-[ -z "$QUTE_URL" ] && QUTE_URL='http://google.com'
 
-url=$(echo "$QUTE_URL" | cat - "$QUTE_CONFIG_DIR/quickmarks" "$QUTE_DATA_DIR/history" | dmenu -l 15 -p qutebrowser)
+[ -z "$QUTE_URL" ] && QUTE_URL='https://duckduckgo.com'
+
+url=$(printf "%s\n%s" "$QUTE_URL" "$(sqlite3 "$QUTE_DATA_DIR/history.sqlite" 'select url from CompletionHistory')" | cat "$QUTE_CONFIG_DIR/quickmarks" - | dmenu -l 15 -p qutebrowser)
 url=$(echo "$url" | sed -E 's/[^ ]+ +//g' | grep -E "https?:" || echo "$url")
 
 [ -z "${url// }" ] && exit
 
 echo "open $url" >> "$QUTE_FIFO" || qutebrowser "$url"
+
```

Comment:
You added empty an empty line at the end of the script.

Reply:
whoops, thats true.